docs/configuration/logs.md

```diff
@@ -146,19 +148,26 @@ format = "json"
 
   [accessLog.filters]
 
-  # statusCodes keep access logs with status codes in the specified range
+  # statusCodes: keep access logs with status codes in the specified range
   #
   # Optional
   # Default: []
   #
   statusCodes = ["200", "300-302"]
 
-  # retryAttempts keep access logs when at least one retry happened
+  # retryAttempts: keep access logs when at least one retry happened
   #
   # Optional
   # Default: false
   #
   retryAttempts = true
+
+  # minDuration: keep access logs when request took longer than the specified duration
```

Comment:
keep access logs when a request takes longer than the specified duration

Reply:
I find confusing to use present form (takes longer) and past form (has ocurred) in the descriptions across filters.

We should probably attempt to get some consistency. Ideas?